### ISSUE TITLE
Be a bit techy verbose on played tracks 

### ISSUE THREAD

**Issue by systems-rebooter** [devsnd/cherrymusic#534]
It would be cool to see some tech info of played tracks with cherry-m, like bitrate, codec, stereo mode, frequency in player slide-bar or in any other place.

**Comment by 6arms1leg** [comment]
I'd like that, too (+ indication if transcoding is `on`). :+1:

**Comment by devsnd** [comment]
hey systems-rebooter,

nice to see CM has grabbed your interest!

I agree that a transcoding indicator would be very useful. This is easy to implement: If the jplayer plays from a source that starts with `/trans/` then it is transcoded, otherwise it is not. This could be [implemented in the playlistmanager. This function](https://github.com/devsnd/cherrymusic/blob/devel/res/js/playlistmanager.js#L697) decides whether a track should be transcoded or not and is called whenever the jplayer is instructed to play any media.

I however don't see any advantage in displaying the bitrate to the user, especially when the music is not transcoded. It only clutters up the interface without any additional value. 

"The bitrate is 128kbit/s", yeah, but what are you going to do about it, now that you know?

**Comment by vasyl-shumskyi** [comment]
`"The bitrate is 128kbit/s", yeah, but what are you going to do about it, now that you know?`
Filter? :)

**Comment by systems-rebooter** [comment]
`"The bitrate is 128kbit/s", yeah, but what are you going to do about it, now that you know?`
Filter? :)

**Comment by tilboerner** [comment]
> Filter

Hmm? So this is about more than just *displaying* some encoding metadata. :) What exactly do you want to do, and why? Underspecified features are hard to implement. :crystal_ball: 

Keep in mind that CherryMusic isn't necessarily bound to the encoding details of the actual audio files; as @devsnd pointed out, the audio that gets played could very well be transcoded.

**Comment by systems-rebooter** [comment]
Nope. I'm for displaying only. Filter would be over-enhancement. So this was just a guess, which doesn't need to be coded. I mean the following: if let's say search gives back 2 or X identical tracks (which resides in different albums), I'd prefer the one with higher bitrate :)
